piker/config.py

```diff
@@ -427,7 +427,7 @@ def load_accounts(
 
 ) -> bidict[str, str | None]:
 
-    conf, path = load()
+    conf, path = load(touch_if_dne=True)
```

Comment:
Longer term I wonder if we should change the name of this flag to something like `cpy_from_template_if_dne: bool` or something.

**Not** something we have to change in this PR, just rubber ducking.